### ISSUE TITLE
Improve type ahead search experience for Screenreader users

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#11089]
## Issue Description
How might we improve the Facility Locator search experience for Veterans using a screenreader?

## Current functionality 
- When letters are entered into the typeahead fields, they are read aloud. The screenreader talks over itself to  present the potential matches displayed in the drop down. 
- Two search fields use typeahead functionality: 

1. Geographic location (City, state, zip)
2. "Service type" when Facility type = community care providers (in VA's network)

This needs to be addressed when we add "Did you mean" suggestions for location. 

## Front end tasks
- Determine potential options to improve this experience. 
- Ensure unit tests are available. 
- Test with 1 or 2 screen readers

## Acceptance Criteria
- [ ] Unit tests pass
- [ ] All screenreader checks pass
- [ ] Veterans using a screenreader during a Facility Locator search can clearly understand what they have entered into a search field. 
- [ ] Veterans using a screenreader during a Facility Locator search can differentiate between their input and the presentation of the drop down matches.  
- [ ] Veterans using a screenreader during a Facility Locator search clearly understand the presentation of the drop down matches.  
- [ ] Veterans can edit their input and understand what has been entered into the search field. 

## Appendix
<details> 

 <summary>Ensure your code changes are covered by E2E tests (expand)</summary>
 
 - Add E2E tests if none exist for this addition/change. 
 
 - Update existing E2E tests if this code will change functionality. 
 
 - Include axe checks, including hidden content
 
</details>

<details> 

 <summary> Run axe checks using the Chrome or Firefox browser plugin (expand)</summary>
 
- Ensure no heading levels are skipped.

- Ensure all buttons and labeled inputs use semantic HTML elements.

- Ensure all buttons, labeled elements and images are identified using HTML semantic markers or ARIA roles.

- Ensure form fields have clearly defined boundaries or outlines.

- Ensure form fields do not use placeholder text. 

- Ensure form fields have highly visible and specific error states. 

</details>

<details> 

 <summary> Test for color contrast and color blindness issues (expand) </summary>
 - All text has appropriate contrast. 

</details>

<details> 

 <summary> Zoom layouts to 400% at 1280px width (expand)</summary> 
 - Ensure readability and usability are supported when zoomed up to 400% at 1280px browser width
 - Ensure no content gets focused offscreen or is hidden from view.
 
</details>

<details> 

 <summary> Test with 1 or 2 screen readers (expand)</summary> 
 
 - Ensure the page includes a skip navigation link. 
 
 - Ensure all links are properly descriptive. 
 
 - Ensure screen reader users can hear the text equivalent for each image conveying information. 

 - Ensure screen reader users can hear the text equivalent for each image button. 

 - Ensure screen reader users can hear labels and instructions for inputs. 
 
 - Ensure purely decorative images are not announced by the screenreader. 
 
</details>

<details> 

 <summary>Navigate using the keyboard only (expand)</summary> 
 
 - Ensure all links (navigation, text and/or image), form controls and page functions can be reached with the tab key in a logical order. 
 
 - Ensure all links (navigation, text and/or image), form controls and page functions can be triggered with the spacebar, enter key, or arrow keys.
 
 - Ensure all interactive elements can be reached with the tab key in a logical order 
 
 - Ensure all interactive elements can be triggered with the spacebar, enter key, or arrow keys.
 
 - Ensure focus is always visible and appears in logical order.
 
 - Ensure each interactive element has visible focus state which appears in logical order.
 

</details>

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by mmiddaugh** [comment]
Will be informed by [RESEARCH] Mobile search expectations (assistive tech) #18685 in Q121

**Comment by mmiddaugh** [comment]
This issue will be addressed during Q3 in Close accessibility gap #26954

**Comment by mmiddaugh** [comment]
This will need to be addressed as part of a much larger Community care initiative
